docs/nodes/credentials/Dropbox/README.md

```diff
@@ -1,16 +1,19 @@
 # Dropbox
 
-You can find information about the operations supported by the Dropbox node on the [integrations](https://n8n.io/integrations/n8n-nodes-base.dropbox) page. You can also browse the source code of the node on [Github](https://github.com/n8n-io/n8n/tree/master/packages/nodes-base/nodes/Dropbox).
+You can find information about the operations supported by the Dropbox node on the [integrations](https://n8n.io/integrations/n8n-nodes-base.dropbox) page. You can also browse the source code of the node on [GitHub](https://github.com/n8n-io/n8n/tree/master/packages/nodes-base/nodes/Dropbox).
 
 ## Pre-requisites
 
 Create a [Dropbox](https://www.dropbox.com/) account.
 
 ## Using OAuth
 
-1. Access this [Dropbox application creation site](https://www.dropbox.com/developers/apps/create).
-2. Fill out the application information.
-3. Click on "Create App".
-4. Use provided credentials with your Dropbox node credentials in n8n.
+1. Access the [Dropbox app creation page](https://www.dropbox.com/developers/apps/create).
+2. Select Dropbox API and scoped access.
+3. Select access type, enter the app name and accept the terms and conditions.
+4. Click on Create App.
+5. In the Dropbox app dashboard, navigate to Settings, scroll down to OAuth2 and click Generate under Access token.
```

Comment:
Could you change it to: click on the 'Generate' button under 'Generated access token'